nemoguardrails/library/autoguard/actions.py

```diff
@@ -78,16 +78,21 @@
 }
 
 
-def process_autoguard_output(response: Any):
+def process_autoguard_output(responses: List[Any]):
     """Processes the output provided AutoGuard API"""
-    if response["task"] == "text_toxicity_extraction":
-        output_str = (
-            GUARDRAIL_RESPONSE_TEXT[response["task"]]
-            + " Toxic phrases: "
-            + " ".join(response["output_data"])
-        )
-    else:
-        output_str = GUARDRAIL_RESPONSE_TEXT[response["task"]]
+
+    prefix = []
+    suffix = []
+    for response in responses:
+        if response["task"] == "text_toxicity_extraction":
+            suffix += response["output_data"]
+        prefix += [GUARDRAIL_RESPONSE_TEXT[response["task"]]]
+
+    output_str = (
+        ", ".join(prefix) + " has been detected by AutoGuard; Sorry, can't process."
+    )
+    if len(suffix) > 0:
```

Comment:
we dont need this; this should be optional - if they want to extract that then can fetch that field in the response. readme should have the formal output format for them

Reply:
resolved it in subsequent commits